fe/fe-core/src/main/java/org/apache/doris/nereids/rules/exploration/join/InnerJoinLAsscomProject.java

```diff
@@ -98,7 +98,7 @@ public Rule build() {
                     newTopHashConjuncts.forEach(expr -> topUsedExprIds.addAll(expr.getInputSlotExprIds()));
                     newTopOtherConjuncts.forEach(expr -> topUsedExprIds.addAll(expr.getInputSlotExprIds()));
                     Plan left = CBOUtils.newProject(topUsedExprIds, newBottomJoin);
```

Comment:
why not apply this for both sides? Means left side's new project is a must?

Reply:
we need a project between join and join